enhancements/ingress/set-delete-http-headers.md

```diff
@@ -735,26 +735,12 @@ another [option](https://docs.openshift.com/container-platform/4.12/networking/i
 but have actual use-cases (which aren't covered by the existing option). They will be allowed, with notes in the API and product documentation,
 and possibly a warning on admission.
 
-For example: If someone creates an Ingress Controller with the following:
-```sh
-apiVersion: operator.openshift.io/v1
-kind: IngressController
-metadata:
-  name: default
-  namespace: openshift-ingress-operator
-spec:
-  httpHeaders:
-    forwardedHeaderPolicy: Replace
-```
-Or if someone add a route annotation `haproxy.router.openshift.io/set-forwarded-headers`,
-then the `set-header` stanzas will be added in `haproxy.config` using the [haproxy-config.template](https://github.com/openshift/router/blob/0cd9a58dff77127562f7851ccfa5cda79b8cef48/images/router/haproxy/conf/haproxy-config.template#L565-L580)
+In the case of HTTP request headers, the actions specified in `spec.httpHeaders.actions` on the Route will be executed after
+the actions specified in the IngressController's `spec.httpHeaders.actions` field.
```

Comment:
I think you need to mention the difference between request and response overrides here, instead of in the questions section.

Reply:
i did not get this. can you please explain more ?

Reply:
Is it true that this only matters for request headers, or is there also some special behavior to describe for response headers?   If needed, I think you should point to the  details that are listed in the section "How do Route headers interact with IngressController headers of the same value".  But I wish there was a brief way to state that here.  But maybe this is not needed for x-forwarded-headers?

Reply:
X-Forwarded headers are request headers so we don't need to describe about response headers from what I understand.